pkg/controller/webhooks/default_handler.go

```diff
@@ -0,0 +1,263 @@
+package webhooks
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"github.com/vshn/appcat/v4/pkg/common/quotas"
+	"github.com/vshn/appcat/v4/pkg/common/utils"
+	"github.com/vshn/appcat/v4/pkg/comp-functions/functions/common"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+type DefaultWebhookHandler struct {
+	client    client.Client
+	log       logr.Logger
+	withQuota bool
+	obj       runtime.Object
+	name      string
+	gk        schema.GroupKind
+	gr        schema.GroupResource
+	length    int
+}
+
+var _ webhook.CustomValidator = &DefaultWebhookHandler{}
+
+// SetupWebhookHandlerWithManager registers the validation webhook with the manager.
+func New(mgrClient client.Client, logger logr.Logger, withQuota bool, obj runtime.Object, name string, gk schema.GroupKind, gr schema.GroupResource, length int) *DefaultWebhookHandler {
+
+	return &DefaultWebhookHandler{
+		client:    mgrClient,
+		log:       logger,
+		withQuota: withQuota,
+		obj:       obj,
+		gk:        gk,
+		gr:        gr,
+		length:    length,
```

Comment:
Can we brainstorm this parameter? I understand that for each service the limit is different but we need to figure out a way that all services get the same limit.

Reply:
Yes we should talk about this